distribution/docs/src/main/resources/content/_running/cql-syntax.adoc

```diff
@@ -0,0 +1,21 @@
+:title: CQL Syntax
```

Comment:
Is it easy to force `Console Command Help` to be the first subsection under `Console Commands` instead of `CQL Syntax`?

Reply:
Yep. done